### ISSUE TITLE
USER ISSUE: 0.8.1.4 => 0.8.2.8 server migration

### ISSUE THREAD

**Issue by salceson666** [StrangeLoopGames/EcoIssues#12767]
Hi, I was able to sucesfully migrate server storage through 0.7.6.3 to 0.8.1.4.
However now I'm stuck and unable to push updates further.
All version above 0.8.1.4 (starting from 0.8.2.0) are getiing same error.

Details and log are below:

Regards.

**Version:** 0.8.2.8 beta

**Steps to Reproduce:**
Load game from 0.8.1.4

**Expected behavior:**
Server started

**Actual behavior:**
The following error message popped up:
```
Connection Failed

Failed to connect - connection timed out.
```

**Do you have mods installed? Does issue happen when no mods are installed?:**No**
The following error message popped up:
```

Failed to connect - connection timed out.

`[22:17:28.874] [1] [ Info] [Eco] >>> START EcoServer, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null 
[22:17:28.896] [1] [ Info] [Eco] Eco Server 0.8.2.8 beta
 
[22:17:28.999] [1] [ Info] [Eco] Server Initialization...  
[22:17:28.999] [1] [ Info] [Eco] Starting LocalizationPlugin...  
[22:17:29.283] [1] [ Info] [Eco] Starting ModKitPlugin...  
[22:17:29.283] [1] [ Info] [Eco] Loading mods...  
[22:17:29.359] [1] [ Info] [Eco] Compiling mods...  
[22:17:33.345] [1] [ Info] [Eco] Loading Eco.IntegrationTests...
 
[22:17:33.376] [1] [ Info] [Eco] Loading mods finished in 4,090,6ms
 
[22:17:33.376] [1] [ Info] [Eco] Starting NetworkManager...  
[22:17:33.376] [7] [ Info] [Eco] Initializing Simulation Types...  
[22:17:33.407] [8] [ Info] [Eco] Starting WorldGeneratorPlugin...  
[22:17:33.407] [10] [ Info] [Eco] Starting GameplayStatsPlugin...  
[22:17:33.407] [6] [ Info] [Eco] Starting BackupPlugin...  
[22:17:33.423] [6] [ Info] [Eco] Starting StorageManager...  
[22:17:34.504] [7] [ Info] [Eco] Initializing items...  
[22:17:36.210] [6] [ Info] [Eco] Starting PausePlugin...  
[22:17:36.210] [6] [ Info] [Eco] Loading Pause...  
[22:17:36.885] [7] [ Info] [Eco] Initializing skills...  
[22:17:36.947] [6] [ Info] [Eco] Starting StatDatabase...  
[22:17:36.947] [8] [ Info] [Eco] Starting WorldPlugin...  
[22:17:36.978] [8] [ Info] [Eco] Loading World...  
[22:17:38.306] [8] [ Info] [Eco] Converting world to new format...  
[22:17:40.699] [8] [ Info] [Eco] Initializing world...  
[22:17:40.824] [8] [ Info] [Eco] Starting TimePlugin...  
[22:17:40.824] [8] [ Info] [Eco] Loading Time...  
[22:17:40.824] [8] [ Info] [Eco] Starting ReputationPlugin...  
[22:17:40.824] [8] [ Info] [Eco] Loading Reputation...  
[22:17:40.840] [8] [ Info] [Eco] Starting PlayerHousingManager...  
[22:17:40.840] [8] [ Info] [Eco] Starting UserManager...  
[22:17:40.856] [8] [ Info] [Eco] Loading Users...  
[22:17:41.068] [8] [ Info] [Eco] Initializing inventories...  
[22:17:41.090] [8] [ Info] [Eco] Starting EcoObjectManager...  
[22:17:41.090] [8] [ Info] [Eco] Loading Objects...  
[22:17:41.168] [8] [ Info] [Eco] Starting GarbagePlugin...  
[22:17:41.168] [15] [ Info] [Eco] Starting DifficultySettings...  
[22:17:41.168] [14] [ Info] [Eco] Starting DisasterPlugin...  
[22:17:41.168] [22] [ Info] [Eco] Starting ControllerManager...  
[22:17:41.168] [8] [ Info] [Eco] Loading Garbage...  
[22:17:41.168] [11] [ Info] [Eco] Starting VoiceManager...  
[22:17:41.168] [20] [ Info] [Eco] Starting EconomyPlugin...  
[22:17:41.168] [19] [ Info] [Eco] Starting ChatServer...  
[22:17:41.199] [20] [ Info] [Eco] Loading Economy...  
[22:17:41.308] [20] [ Info] [Eco] Starting Legislation...  
[22:17:41.308] [14] [ Info] [Eco] Loading Disasters...  
[22:17:41.308] [19] [ Info] [Eco] Loading Chat...  
[22:17:41.808] [20] [ Info] [Eco] Loading Laws...  
[22:17:41.855] [20] [ Info] [Eco] Starting PropertyManager...  
[22:17:41.855] [20] [ Info] [Eco] Loading Property...  
[22:17:41.902] [20] [ Info] [Eco] Starting MinimapManager...  
[22:17:41.902] [20] [ Info] [Eco] Starting EcoSim...  
[22:17:41.961] [20] [ Info] [Eco] Loading Plants...  
[22:17:42.263] [1] [ Info] [Eco] Server Initialization finished in 13,267,7ms
 
[22:17:42.263] [1] [ Info] [Eco] Failed to load server, Exception was Exception: Exception
Message:Error(s) occurred when loading Plants
Source:Eco.Core

System.Exception: Error(s) occurred when loading Plants
   at Eco.Core.Serialization.DataStore.LoadAsync[T](String folder, Action`2 createFunc)
   at Eco.Simulation.PlantSim..ctor()
   at Eco.Simulation.EcoSim.Initialize()
   at Eco.Simulation.EcoSim..ctor()
 
`
```

**Comment by SlayksWood** [comment]
@salceson666 
Please provide your save files.
Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Uploading-a-world-save

**Comment by salceson666** [comment]
@SlayksWood thanks for reply, game.eco uploaded:
https://wetransfer.com/downloads/9420502809ae2b7d18c53917693d5f2e20190717070928/2f57d9484d4e26a734fee8cbae6f42b620190717070928/f73b72

**Comment by mirasrael** [comment]
@salceson666 I fixed error in code (fix will be available in 8.2.9), but if you don't want to wait - there is fixed save
[Game.zip](https://github.com/StrangeLoopGames/EcoIssues/files/3401015/Game.zip)

**Comment by salceson666** [comment]
@mirasrael save loaded sucesfully - many thanks!
BR
